internal/pkg/encoding/json/schema/schema.go

```diff
@@ -133,7 +133,7 @@ func processErrors(errs []*jsonschema.ValidationError, parentIsSchemaErr bool) e
 		isSchemaErr := !strings.HasPrefix(e.AbsoluteKeywordLocation, pseudoSchemaFile)
 		path := strings.TrimLeft(e.InstanceLocation, "/")
 		path = strings.ReplaceAll(path, "/", ".")
-		msg := strings.ReplaceAll(e.Message, `'`, `"`)
+		msg := strings.ReplaceAll(strings.ReplaceAll(e.Message, `'`, `"`), `n"t`, `n't`)
```

Comment:
Fix `don"t` -> `don't`, asi to zatial nebudem komplikovat nejakym regex vyrazom.